packages/manager/src/components/EditableText/EditableText.tsx

```diff
@@ -192,7 +197,11 @@ export const EditableText = (props: PassThroughProps) => {
     }
   };
   const labelText = (
-    <H1Header className={classes.root} data-qa-editable-text title={text} />
+    <H1Header
+      className={classes.root}
+      data-qa-editable-text
+      title={`${text} ${textSuffix || ''}`}
```

Comment:
It looks like the space character between `text` and `textSuffix` is causing the DBaaS test failures.

```suggestion
      title={`${text}${textSuffix ? ` ${textSuffix}` : ''}`}
```
Confirmed that this gets the tests passing, even though the nested string literal is kinda nasty

Reply:
Oh good catch actually we don't want a whitespace at all. (should be handled in the `textSuffix` as needed by the developer). Fixed in https://github.com/linode/manager/pull/10096/commits/f1e65fc052908b711ad1f3c1ae7b43e2133eac84